apps/site-projects/src/components/modules/Resources/ProjectResources/TeamLinks/ResourceCard.js

```diff
@@ -0,0 +1,15 @@
+import Image from "next/image";
+import Button from "../../../../../../../../packages/UI/src/components/atoms/Button";

```

Comment:
please use imports this way.
import { Button } from "@devlaunchers/components/src/components/atoms";